catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -336,7 +342,8 @@ public void deleteRegistryEntriesByMetacardIds(List<String> metacardIds,
             if (!deleteResponse.getProcessingErrors()
                     .isEmpty()) {
                 throw new FederationAdminException(
-                        "Processing error occurred while deleting registry entry.");
+                        "Processing error occurred while deleting registry entry. Details\n"
```

Comment:
Should we use something like `System.lineSeparator()` that is more platform friendly rather than `\n`?